scripts/automation/Radius/Functions/Private/SystemTable/New-SystemTable.ps1

```diff
@@ -0,0 +1,34 @@
+function New-SystemTable {
+    [CmdletBinding()]
+    param (
+        [Parameter()]
```

Comment:
Add mandatory

Reply:
resolved in 8640140f586990a96984a715295349976a0160bc